java/bulk-import/bulk-import-starter/src/main/java/sleeper/bulkimport/starter/executor/EmrServerlessPlatformExecutor.java

```diff
@@ -87,15 +97,13 @@ public void runJobOnPlatform(BulkImportArguments arguments) {
                 .name(jobName)
                 .executionRoleArn(
                         instanceProperties.get(BULK_IMPORT_EMR_SERVERLESS_CLUSTER_ROLE_ARN))
-                .jobDriver(JobDriver.builder()
-                        .sparkSubmit(SparkSubmit.builder()
-                                .entryPoint("/workdir/bulk-import-runner.jar")
-                                .entryPointArguments(
-                                        instanceProperties.get(CONFIG_BUCKET),
-                                        bulkImportJob.getId(), taskId, arguments.getJobRunId())
-                                .sparkSubmitParameters(constructSparkArgs(instanceProperties))
-                                .build())
-                        .build())
+                .jobDriver(JobDriver.builder().sparkSubmit(SparkSubmit.builder()
+                        .entryPoint("/workdir/bulk-import-runner.jar")
+                        .entryPointArguments(instanceProperties.get(CONFIG_BUCKET),
+                                bulkImportJob.getId(), taskId, arguments.getJobRunId())
+                        .sparkSubmitParameters(
+                                constructSparkArgs(taskId, arguments, bulkImportJob, platformSpec))
+                        .build()).build())
                 .configurationOverrides(
                         ConfigurationOverrides.builder()
                                 .monitoringConfiguration(MonitoringConfiguration.builder()
```

Comment:
Is applicationConfigurations here the right place to use ConfigurationUtils, similar to how it's used in EmrInstanceFleets and PersistentEmrBulkImportStack? Maybe that should be a separate issue?

Reply:
I think maybe a separte issue. It could also be added into the stack, so we need to decide if to add here at the job level and or the application creation

Reply:
Issue raised https://github.com/gchq/sleeper/issues/1384